code/game/objects/items/weapons/scissors.dm

```diff
@@ -0,0 +1,109 @@
+/obj/item/weapon/scissors
+	name = "Scissors"
+	desc = "Those are scissors. Don't run with them!"
+	icon_state = "scissor"
+	item_state = "scissor"
+	force = 5
+	sharp = 1
+	edge = 1
+	w_class = 2
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("slices", "cuts", "stabs", "jabs")
+
+/obj/item/weapon/scissors/barber
+	name = "Barber's Scissors"
+	desc = "A pair of scissors used by the barber."
+	icon_state = "bscissor"
+	item_state = "scissor"
+	attack_verb = list("beautifully slices", "artistically cuts", "smoothly stabs", "quickly jabs")
+
+/obj/item/weapon/scissors/attack(mob/living/carbon/M as mob, mob/user as mob)
+	if(user.a_intent != "help")
+		..()
+		return
+	if(!(M in view(1))) //Adjacency test
+		..()
+		return
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		//see code/modules/mob/new_player/preferences.dm at approx line 545 for comments!
+		//this is largely copypasted from there.
+		//handle facial hair (if necessary)
+		var/list/species_facial_hair = list()
+		if(H.gender == MALE || H.get_species() == "Vulpkanin")
```

Comment:
In the future, we should probably implement some generic system for species to be able to have facial hair in both genders.